File: Python/3342.py
tamanho = int(input())

tamanho *= tamanho

verifica_se_par = True if tamanho%2==0 else False

if verifica_se_par: #se for par, o número de casas brancas e pretas será igual
    print(f"{tamanho//2} casas brancas e {tamanho//2} casas pretas")
else: #se for ímpar, o número de casas brancas será maior em uma unidade
    print(f"{(tamanho//2)+1} casas brancas e {tamanho//2} casas pretas")

File: C/3342.c
#include <stdio.h>

int main() {
    int tamanho, white, black;
    scanf ("%d", &tamanho);
    tamanho = tamanho * tamanho;
    if (tamanho%2==0){
        white = (tamanho)/2;
        printf ("%d casas brancas e %d casas pretas\n", white, white);

    } else {
        white = tamanho/2 +1;
        black = white - 1;
        printf ("%d casas brancas e %d casas pretas\n", white, black);
    }

    return 0;
}